### PR TITLE
chore: Add test image build configuration

### PR DESCRIPTION
**Pull request by dsimansk**
/assign @navidshaikh 

Depends on https://github.com/openshift/knative-client/pull/446

### CODE REVIEW

**Comment by dsimansk** [comment]
We need to sort out prior release that doesn't require this change.

/hold

**Review by navidshaikh — APPROVED** [review]
/lgtm
/approve

File: ci-operator/config/openshift/knative-client/openshift-knative-client-release-next__45.yaml [hunk 1]
```diff
@@ -34,6 +34,14 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/knative/client/package_cliartifacts.sh
   to: kn-cli-artifacts
+- dockerfile_path: openshift/ci-operator/knative-test-images/helloworld/Dockerfile
+  from: base
+  inputs:
+    bin:
```
Comment (navidshaikh) [inline ci-operator/config/openshift/knative-client/openshift-knative-client-release-next__45.yaml:40]:
```suggestion
    test-bin:
```

**Comment by dsimansk** [comment]
The test job is failing due to required setup changes from https://github.com/openshift/knative-serving/pull/539.

**Review by navidshaikh — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**